### DISCUSSION TITLE
Ryzen 5 4600G

### DISCUSSION THREAD

**Discussion by CraigHazan** [ChefKissInc/NootedRed#115] [General]
Hi everyone, happy to be part of this. I just built my first Ryzentosh with a Ryzen 5 4600G. I paired the APU with a ASUS Prime A320I-K motherboard. I had a copy of Ventura 13.3 installed on a spare NVMe and created a new EFI with NootedRed. Success I think, I'm not really sure what I'm supposed to be looking for to see what kind of acceleration I have, but everything seems smooth so far.

![Screenshot 2023-07-04 at 7 50 55 PM](https://github.com/NootInc/NootedRed/assets/20331487/37532a8e-09d9-4f73-832f-7bc90aacd6b4)
![Screenshot 2023-07-04 at 6 50 16 PM](https://github.com/NootInc/NootedRed/assets/20331487/4ec7dd27-c5cb-4d0d-981d-d32a8b11eb64)

**Comment by coimbrox** [comment]
hey man, acceleration is good? for real?  send me a invite to discord for teach me make this coimbrox is my nick

**Reply by VisualEhrmanntraut** [reply]
https://nootinc.github.io/

**Comment by CraigHazan** [comment]
Not quite!, I did recently update from Ventura 13.3 to 13.4.1 and wonder if that's what's causing this, bluetoothhd seems to be the culprit. it just loops from here and doesn't boot.

![verbosehd](https://github.com/NootInc/NootedRed/assets/20331487/f3ff9fe6-10e8-47fd-957b-74c14b77bbd3)

**Reply by jotalegal** [reply]
maybe your smbios doesnt support the new version or maybe your efi folder miss some configuration, try to reinstall macos ventura 13.3 or re-do all the ryzentosh process.

**Reply by jotalegal** [reply]
or you can also send me the efi to i look/see

**Reply by CraigHazan** [reply]
I'm using SMBIOS iMacPro1,1 works well for me.

**Reply by VisualEhrmanntraut** [reply]
This is a Lilu bug. It does not detect the last stage of installation/update and that makes NRed load personalities for kexts that don't exist. Complain to Acidanthera for neglecting my issue for months

**Comment by truongvantuan** [comment]
Hi bro, VRAM with 2Gb is default or get by patch method?

**Reply by jotalegal** [reply]
the recommended one is 512mb, 1 or 2gb are recommended for video editing and etc

**Reply by CraigHazan** [reply]
You can change it in BIOS, I'm on a ASUS Prime A320I-K, with 8 GB RAM installed I can allocate 4G to VRAM. with 16 GB RAM I can allocate 8 GB to VRAM. On a Gigabyte B450 with 8 GB RAM installed I could allocate 16 GB VRAM!.

**Reply by truongvantuan** [reply]
My bios setting just have 265, 512, Auto.

**Reply by CraigHazan** [reply]
How much RAM do you have installed?. 512 is the minimum for NRed, Auto might be worth trying.

**Reply by truongvantuan** [reply]
I have try Auto mode, but still get512 VRAM

**Reply by CraigHazan** [reply]
I've noticed the more RAM you install, more VRAM is available for the Frame Buffer. If I install 1 x 8 GB RAM I have access to up to 4 GB VRAM. If I install 2 x 8 GB RAM I have access to up to 8 GB VRAM. But, if I install 8 GB RAM into a Gigabyte 450 motherboard I have access to up to 16 GB VRAM.

**Comment by CraigHazan** [comment]
It's all good!, I refreshed my EFI and we're back in business. Updated to 13.4.1 (c) without any issues, just like a regular Mac. Just installed the latest build 33e9abf looks better than before.

![Screenshot 2023-07-15 at 10 33 33 AM](https://github.com/NootInc/NootedRed/assets/20331487/87869cc8-7a04-4818-9c5d-59cb319bbc67)

**Comment by CraigHazan** [comment]
Successfully installed macOS Sonoma, using MacPro7,1 SMBIOS which I assume is working now.

![Screenshot 2023-07-18 at 11 15 41 PM](https://github.com/NootInc/NootedRed/assets/20331487/ea0b20fd-e70d-4dfc-a42b-81b2a203c6b8)

**Reply by tioburla** [reply]
Hi, i'm using R4600G + RTX 3050, MotherBoard A520 MSI and 16GB RAM 3000Mhz
U have any idea if works fine with Grapichs Card? Or only with Vega Graphics?
